lib/snapshot/snapshot.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// ignore_for_file: unused_import,uri_does_not_exist
+
```

Comment:
I'd rather we used explicit `// ignore` on each line rather than `ignore_for_file`. `ignore_for_file` makes it very easy to add new problems. If we really want to just turn it off, we can do so in the analysis_options file or using `grep -v` or some such.

Reply:
I'd like to add `native_function_body_in_non_sdk_code: ignore` to analysis options then - There are a lot of places it gets used, and it would have to be remembered by anyone who adds a native binding method to lib/ui.  Thoughts on that?

Reply:
For instance, painting.dart has something like 100 lines that use the `native` keyword.